---
content_type: page
description: This section contains lecture summaries for every week.
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 7c050027-bd00-bba9-6a48-cfe169c3dcb4
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
WEEK #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction to the human microbiome
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week1" >}}{{< /anchor >}}This first class is dedicated to an introduction of the course, students and lecturers. We will go over the syllabus, goals and expectations for the course and schedule meeting times. The instructors will introduce the core concepts of the microbiome, cross-species signaling, synthetic biology and therapeutics. This introduction will provide an overview of the exciting field of the microbiome and engineering microorganisms for drug delivery applications.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen colspan="3" >}}
**Small-molecule Signals, Microfluidics and Synthetic Biology**
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Microbes use small molecules to communicate
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week2" >}}{{< /anchor >}}To navigate their complex environments, microbes use chemical cues to communicate with each other. One common type of communication called "quorum sensing" uses small molecules secreted and sensed by microbes of the same species to determine whether a "quorum" has been reached. This communication enables the bacteria to act in unison when a substantial number of the same bacteria are present. Certain bacteria, such the opportunistic human pathogen _Pseudomonas aeruginosa_, use quorum sensing signaling to coordinate the expression of virulence genes thereby giving the bacteria strength in numbers against the host and their environment. In this session we will focus on the mechanistic underpinnings of the quorum sensing system found in _P. aeruginosa_.

**Video:** TED-Ed. "{{% resource_link "e74ec50b-c649-4bca-93d7-76001a93cd06" "How bacteria 'talk' - Bonnie Bassler" %}}." February 9, 2013. YouTube.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Microfluidics technology to study the human microbiome
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week3" >}}{{< /anchor >}}Microfluidics is the science and technology of manipulation and precise control of fluids geometrically constrained to a sub-millimeter scale. This technology enables recapitulating the human gut microbiome "on a chip." Because of versatile modularity for mimicking the intestinal pathophysiology and more refined analysis of bacterial–bacterial and bacterial–epithelial cell signalling in a laboratory environment, this method promises to revolutionize therapeutic strategies for a variety of diseases.

In the first paper by Luo et al., students will learn about engineering an organ on a chip system to quantitatively evaluate the synthesis and consumption dynamics of signalling molecules and cell responses.

The application of microfluidic technology is not limited in "organ on a chip" systems. In the second paper, by Hong et al., class participants learn about designing a synthetic quorum-sensing circuit to control consortial biofilm formation and dispersal in a microfluidic device.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Field trip to {{% resource_link "65f4c142-c3f5-4a4b-b862-d4df8a00c314" "Ginkgo Bioworks" %}}, the organism company
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week4" >}}{{< /anchor >}}The field of synthetic biology or genetic engineering of cells, including microbes, has grown beyond academic research. Companies are applying the tools of genetic engineering to develop products for a diverse range of applications, including human health. {{% resource_link "65f4c142-c3f5-4a4b-b862-d4df8a00c314" "Ginkgo Bioworks" %}} is a company founded by MIT alumni in 2009 that aims to make biology engineerable. Ginkgo Bioworks has established infrastructure, modelled after automated microchip foundries, which aims to generate large number of reproducibly engineered microbes. An early perspective by Drew Endy provides an overview of how groups such as those at Ginkgo Bioworks are attempting to engineer biology through the use of standardized parts. A more recent perspective by Roberta Kwok discusses the unavoidable challenges of engineering biology and asks whether it is in fact possible as originally outlined. This week we will take a field trip to visit the Ginkgo Bioworks foundry in the drydocks area, where students can learn about and ask how this leading company is tackling the challenges of translating the results found in academic literature into a commercial enterprise. Will such companies be able to leverage microbiome signaling to develop therapeutics?
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Microbes engineered to detect and potentially kill pathogens
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week5" >}}{{< /anchor >}}Recent advances in synthetic biology, such as those being developed at Ginkgo Bioworks, have made it possible to engineer genetically modified biological systems to perform novel functions not existing in nature. In this session we will cover two papers showing the use of engineered microbes. The work of Danino et al., is an elegant example of developing an orally administered diagnostic, capable of noninvasively indicating the presence of liver metastasis by producing easily detectable signals in urine.

The paper by Saeidi at al., describes the development of a synthetic genetic system, composed of quorum sensing, killing, and lysing switches, enabling _Escherichia coli_ to sense and kill a pathogenic _Pseudomonas aeruginosa_ strain through the production and release of pyocin.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen colspan="3" >}}
**Peptide Signals for Defense, Therapeutics and Competence**
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Microbes use peptides as a defense and as signals
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week6" >}}{{< /anchor >}}Lantibiotics are a class of secreted peptides that are produced by gram-positive bacteria to inhibit the growth of other bacteria. Nisin is a well studied lantibiotic that has found use in the food industry to preserve food. In addition, there has been renewed interest in nisin as a signaling molecule, since it was found to control its own expression via a receptor protein that senses nisin and transduces the signal to eventually upregulate nisin production. This self-regulating signaling / defense molecule enables the producing bacteria to secrete nisin only when a "quorum" has been reached, enhancing the total amount and efficacy of nisin as a defense antibiotic. This signaling system is now being harnessed in synthetic biology as a genetic control signal that enables researchers to specifically upregulate the expression of any gene of interest that is placed under the control of a nisin-regulated promoter. In this session, we look at how the antibiotic target of nisin was determined (Brötz 1998), as well as how the signaling system of this secrete peptide was applied to control an engineered genetic system (de Ruyter 1996).
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Drug delivery using microbes engineered to secrete peptides
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week7" >}}{{< /anchor >}}The art of engineering of microbes has made it possible to use living recombinant micro-organisms as live delivery vehicles for compounds of interest in the human digestive tract. This method has several advantages. For example, it allows the administration of drugs known to be sensitive to digestive secretions when given with classical pharmaceutical formulations. This week we will discuss two papers. The first is about engineering bacteria secreting an HIV fusion inhibitor peptide that can be potentially used as a therapy for AIDS patients.

The second paper is about engineering yeast that secrete proteins or peptides as a new drug delivery system in the gut. The authors investigate the survival rate and the ability of two recombinant _Saccharomyces cerevisiae_ strains to initiate the synthesis and secrete either a model peptide or a model protein (glutathione-S-transferase-V), in a simulated gastric-small intestinal system.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Competence-stimulating peptides as cross-species signals
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week8" >}}{{< /anchor >}}Oral microbiome species in the _Streptococcus_ genus secrete peptides that self-induce competence, a state in which the bacterial cells can uptake foreign DNA. Such a Competence Stimulating Peptide (CSP) functions as a quorum sensing molecule and enables the bacteria to coordinate switching into this new state. Recently it was discovered that CSP also functions as a cross-species signal that modulates the growth of _Candida albicans_, a fungus that lives in the same niche as the _Streptococcus_ species. In this session, we look at how CSP was initially discovered and its signaling function validated (Håvarstein 1995) and then look at recent results in support of its cross-species signaling function against _Candida albicans_ (Jarosz 2009).
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen colspan="3" >}}
**The Future of the Microbiome for Human Health**
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Using seminars and talks for sharing scientific discoveries
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week9" >}}{{< /anchor >}}Besides journal papers, seminars and talks are also a great way for sharing scientific discoveries. In this session, we will discuss a brief pre-recorded seminar by Christopher Voigt in an _iBiology_ series on the topic of synthetic biology and programming living organisms for novel applications including drug delivery. Students are expected to watch the video before coming to this session, using the following link:

**Video:** Chris Voigt. "{{% resource_link "73c928be-297b-4a1c-bb98-af1c902b6c7e" "Genetic Circuits: Programming Living Bacteria" %}}." July, 2015. iBiology.

Then in the class, we will discuss the talk, basic concepts and recent advances of the field. We will critically look at the concepts and question the claims promised as future outcomes of the field.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Advance culturing techniques to study microbiomes
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week10" >}}{{< /anchor >}}Two important and advanced methods for studying human microbiome will be introduced and discussed this week: Gnotobiotic mice and human gut on a chip. A gnotobiotic animal is one in which only certain known strains of bacteria / microorganisms are present. Such animals are used to study the interaction between an animal and one or more of the microorganisms that inhabit its body. This technique is very useful, because it allows the study of only a select few symbiotic interactions at a time. In the first paper, Samuel et al. use gnotobiotic mouse to do in vivo analyses of the impact of a methanogen and a saccharolytic bacterium on one another's transcriptomes or metabolomes. Their results indicate that _M. smithii_ regulates the specificity of polysaccharide fermentation and influences the amount of calories deposited in fat stores.

The human-gut-on-a-chip technology is a 3D microfluidic platform that mimics the natural conditions of the human intestine in a small-scale, controllable, _in vitro_ environment. This technology allows the study of microbes over a longer time scale, which has not been possible so far by use of traditional culture methods. In the second paper, Kim et al. show the development of a microengineered (gut on a chip) model of human intestinal inflammation and bacterial overgrowth that allows analysis of individual factors to the pathophysiology of intestinal diseases, such as ileus and inflammatory bowel disease, over couple of weeks _in vitro_.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Host-generated peptides shape their microbiomes
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week11" >}}{{< /anchor >}}Interactions in a microbiome also involve signals secreted by the host. Antimicrobial peptides (AMPs) are one such signal that is secreted by host cells. As their name suggests, these secreted peptides have antimicrobial activity; they also are an important arm of the innate immune system. While these peptides generally have broad antimicrobial activity, recent studies show that this activity specifically selects for the growth of some bacteria while inhibiting the growth of others. In this session, we look at the mechanism by which the secretion of the AMP human defensin 5 is regulated in the intestinal tract (Ghosh 2002). In the second paper, we look at how specific lipids on the cell surfaces of some bacteria enable the bacterial cells to grow even in the presence of AMPs such as defensin 5 and thereby enable these bacteria to become permanent parts of the gut microbiome (Cullen 2015).
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Microbes as neuromodulators
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week12" >}}{{< /anchor >}}Our microbiomes are important modulators of the gut-brain axis. The gut-brain axis is the communication that occurs between the enteric nervous system (gut) and the central nervous system (brain). This suggests our microbiomes can potentially impact not just our GI tract but also our brain and therefore might play a role in shaping how we think and behave. Microbes might have this effect through the modulation of neuroactive compounds, either by directly producing them or by stimulating production in the host. In turn, host processes such as circadian rhythms, which are controlled centrally in the brain, can influence the microbiome, suggesting that there is signaling loop between us and our microbiomes. However, much remains to be studied in this complex area. In this session, we look at how germ-free mice are being used to support the notion that the microbiome can affect brain development (Heijtz 2011) as well as the idea that oscillations in the host such as the circadian rhythm control oscillations in the microbiome (Thaiss 2014).
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Microbes as cancer therapeutics
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week13" >}}{{< /anchor >}}As early as the 19{{< sup "th" >}} century, it was discovered that certain bacterial infections might have a marked therapeutic effect on certain cancer patients. Bacteria can sense their environment and deliver proteins to eukaryotic cells. In this session we talk about engineering bacteria to invade cancer cells. In the first paper, Zhao et al. demonstrate the development of an effective bacterial cancer therapy by targeting tumor tissue, using _Salmonella typhimurium_ leu-arg auxotrophs. This auxotrophy successfully restricts growth of bacteria in normal tissue. These bacteria were used to treat metastatic human prostate tumors implanted in nude mice. They show some of the treated mice were alive and well at the time the last untreated mouse died.

In the second paper Anderson et al. demonstrate engineering of _Escherichia coli_ to invade cancer-derived cell lines including HeLa, HepG2, and U2OS.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
Oral presentations, conclusion and closing remarks
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week14" >}}{{< /anchor >}}Students will give oral presentations (~ 20 min) about their chosen research papers and lead critical discussions about the papers' findings. We will have a final discussion about what we have learned about the human microbiome, signaling, drug delivery, the challenges of the field and future directions. Finally, students will complete course evaluations and provide feedback to the instructors.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}